contracts/test/GrantRound.test.ts

```diff
@@ -158,7 +162,7 @@ describe('GrantRound', function () {
       // TODO: add token.balanceOf check in mock ERC20
     });
 
-    it('reverts if not the grant round payout administrator', async function () {
+    it.skip('reverts if not the grant round payout administrator', async function () {
```

Comment:
I'll be refactoring these tests anyway with the ERC20 mocks but one thing that's trouble some is because the contract now checks the block time stamp when it is deployed (an instance is created before each test), the time travel to a respective grant round instance endTime is now in the past which is problematic for testing. 

Any ideas on how to get around this?

Reply:
The way we should do this is with Waffle fixtures: https://ethereum-waffle.readthedocs.io/en/latest/fixtures.html

Basically:
1. Move the content of the `beforeEach` hook into a method called `setup` or something similar
2. The content of the before each hook should now assign all variables e.g. `({registry, roundContract} = await loadFixture(setup);`
3. Adjust time _foward_ in tests as necessary

You can have other fixture methods to load at the beginning of tests too if that's helpful.

The Waffle fixtures use `evm_snapshot` and `evm_revert` methods to revert to chain's state (include timestamp) before each test. That way at the beginning of each test, time resets -> contract deploys -> time can be fast forwarded again if needed

If you need help with this let me know!